CMakeLists.txt

```diff
@@ -71,43 +71,43 @@ if(BUILD_GDASBUNDLE)
   endif()
 
 # Gibbs seawater
-  ecbuild_bundle( PROJECT gsw GIT "https://github.com/jcsda-internal/GSW-Fortran.git" BRANCH develop )
```

Comment:
Please do not change the `CMakeLists.txt` file in develop

Reply:
@CoryMartin-NOAA I think I used the build from dev/ocnda branch

Reply:
Just checkout develop's version of the `CMakeLists.txt` and you should be good to go

Reply:
I am closing this PR. I need to fix some more things